compiler_opt/tools/extract_ir.py

```diff
@@ -232,19 +275,56 @@ def make_obj(obj_file: str) -> TrainingIRExtractor:
   return [make_obj(obj_file) for obj_file in just_obj_paths]
 
 
+def load_for_lld_thinlto(obj_base_dir: str,
+                         output_dir: str) -> List[TrainingIRExtractor]:
+  cmd = [
+      'find', '.' if obj_base_dir == '' else obj_base_dir, '-name',
+      '*.3.import.bc'
+  ]
+  try:
+    with subprocess.Popen(cmd, stdout=subprocess.PIPE) as p:
```

Comment:
I'd rather we recursively iterated through directory structure using filesystem APIs - rather than spawning a process and parsing output: it's less fragile (less dependent on an external tool, output format, IPC stuff, etc)

pathlib, I think, has some APIs for that.